benches/bench.rs

```diff
@@ -100,6 +113,19 @@ mod bench_tests {
         })
     }
 
+    #[bench]
+    fn bench_sokoban_binary_heap_insert_1000_u128(b: &mut Bencher) {
+        let mut rng = rand::thread_rng();
+        let mut sokoban_heap = Heap::<u128, u128, 1001>::default();
+        let mut slice: Vec<u128> = (0..1000).collect();
+
+        b.iter(|| {
+            for v in 0..1000 {
+                sokoban_heap._push(rng.gen::<u128>());
+            }
+        })
+    }
+
```

Comment:
Can you paste the benchmark stats?

Reply:
test bench_tests::bench_sokoban_binary_heap_insert_1000_u128          ... bench:      44,853 ns/iter (+/- 617)
test bench_tests::bench_sokoban_binary_heap_insert_20000_u128         ... bench:      45,293 ns/iter (+/- 1,492)
test bench_tests::bench_sokoban_binary_heap_remove_1000_u128          ... bench:      91,893 ns/iter (+/- 415)